src/SFA.DAS.ProviderRequestApprenticeTraining.Web.UnitTests/Controllers/HomeControllerTests.cs

```diff
@@ -1,46 +1,57 @@
-﻿using AutoFixture.NUnit3;
-using FluentAssertions;
+﻿using FluentAssertions;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Options;
+using Moq;
 using NUnit.Framework;
 using SFA.DAS.Provider.Shared.UI.Models;
 using SFA.DAS.ProviderRequestApprenticeTraining.Web.Controllers;
-using SFA.DAS.Testing.AutoFixture;
 
-namespace SFA.DAS.ProviderRequestApprenticeTraining.Web.UnitTests.Controllers
+namespace SFA.DAS.ProviderRequestApprenticeTraining.Web.Tests.Controllers
```

Comment:
This should be UnitTests if the project is UnitTests

Reply:
Done